ansible_collections/arista/avd/plugins/plugin_utils/schema/avddataconverter.py

```diff
@@ -194,6 +194,15 @@ def _convert_types(validator, convert_types: list, instance, schema: dict):
                     converted_instance = instance
                     pass
                 return converted_instance
+            elif convert_type == "str" and schema_type == "list":
+                try:
+                    converted_instance = instance.split(",")
```

Comment:
please also .strip() the elements in case there are any spaces around the comma.

Reply:
Updated